shed/tests/test_streams.py

```diff
@@ -454,6 +454,25 @@ def f(img1):
         assert n in assert_docs
 
 
+def test_filter_full_header(exp_db, start_uid1):
+    source = Stream()
+
+    def f(docs):
```

Comment:
to increase the coverage, maybe add a test for a function with the negation of this predicate.
(you're missing one line of coverage)